docs/data/data-grid/components/components.md

```diff
@@ -184,8 +184,10 @@ For example, for `columnMenu` slot, the interface name would be `ColumnMenuProps
 
 This [file](https://github.com/mui/mui-x/blob/-/packages/grid/x-data-grid/src/models/gridSlotsComponentsProps.ts) lists all the interfaces for each slot which could be used for augmentation.
 
-```tsx
-// augment the props for `toolbar` slot
+<codeblock storageKey="data-grid-pricing-plan">
```

Comment:
I was hesitating to say that we don't need a storage key for this, but maybe we do, a general one seems more appropriate.

```suggestion
<codeblock storageKey="pricing-plan">
```

Reply:
Yeah, that was my initial thought, but then I thought it might be better to have a separate key for the data grid and pickers because there's no Premium plan in pickers. With separate storage keys, you can have Pro for the pickers and Premium for the grid.
Not a strong opinion though 😅

Reply:
We could have special handling for this case in the docs-infra.